pkg/synthetictests/platformidentification/types.go

```diff
@@ -155,3 +204,26 @@ func getArchitecture(clientConfig *rest.Config) (string, error) {
 
 	return "amd64", errors.New("could not determine architecture from master nodes")
 }
+
+// HasCompleted returns nil if the cluster has completed the provided
+// major.minor release.  For example, HasCompleted("4.10") returns nil
+// if the cluster has completed 4.10 or a later minor version.  If the
+// cluster has not, HasCompleted returns an error mentioning the most
+// recently completed major.minor.
+func (jt *JobType) HasCompleted(version string) error {
```

Comment:
This is `IsVersionAtLeast`

Reply:
In aee3c43236 -> 6a0d9ba06b, I went with `MostRecentlyCompletedVersionIsAtLeast`, because it's an `error`-returning assertion, not a `bool`-returning query.